feat(coverage): add support for Sonar Findbugs + Scala plugin

skirge opened the pull request:
Changes required by Sonar Findbugs + Scala plugin and some bugfixes done while developing it.

Commit:
fix(coverage): fix sonar.source with semicolons

add support for sonar.sources property containing multiple directories
separated by semicolon

SagaStalin commented:
<KGB>e736cf7e49f6ca1886023c3271dc46fa7335998a
>fix(coverage): add support for sonar.sources property containing multiple directories separated by semicolon

Shorten status from 108 to 50 characters
Wrap body lines at 72 charactersMore info [here](https://github.com/sagacify/komitet-gita-bezopasnosti)

Commit:
fix(coverage): fix for absolute vs relative paths

fix for maven sending absolute paths and sonar-runner sending relative
paths

Commit:
fix(coverage): fix exception on empty coverage

fix exception while running sonar plugin and coverage is generated
for empty module

SagaStalin commented:
<KGB>93d06d02972c3006f2871a6601febe99b35cf3cc
>fix(coverage): fix for absolute vs relative paths
>
>fix for maven sending absolute paths and sonar-runner sending relative paths

Wrap body lines at 72 characters

43887f7205a2617ad0b5982c4a9707587be0fbec
>fix(coverage): fix exception on empty coverage
>
>fix exception while running sonar plugin and coverage is generated for empty module

Wrap body lines at 72 charactersMore info [here](https://github.com/sagacify/komitet-gita-bezopasnosti)

SagaStalin commented:
<KGB>00d1305f80a26826447dff9d7e7cffec16d9c32b
>feat(coverage): move language definition to Java file
>
>move language definition to Java file so it can be referenced from other plugins.
>It didn't work with Scala class (missing Scala runtime on classpath?)
>Package name is consistent with Sonar convention.

Shorten status from 53 to 50 characters
Wrap body lines at 72 charactersMore info [here](https://github.com/sagacify/komitet-gita-bezopasnosti)

Commit:
feat(coverage): move language definition to Java

move language definition to Java file so it can be referenced from
other plugins. It didn't work with Scala class
(missing Scala runtime on classpath?)
Package name is consistent with Sonar convention.
